map/map_tests/bookmarks_test.cpp

```diff
@@ -1387,7 +1388,82 @@ UNIT_CLASS_TEST(Runner, Bookmarks_AutoSave)
   TEST(base::DeleteFileX(fileName2), ());
 }
 
-UNIT_CLASS_TEST(Runner, Bookmarks_BrokenFile)
+UNIT_CLASS_TEST(Runner, ExportAll)
+{
+  string files[] = {
```

Comment:
```suggestion
  std::string const gpxFiles[] = {
```

Reply:
Btw, why GPX? We don't store them internally.

Reply:
In the test later I do:

```
  BookmarkManager::KMLDataCollection kmlDataCollection;
  for (auto const & file : files)
    kmlDataCollection.emplace_back(file, LoadKmlFile(file, KmlFileType::Gpx));
  bmManager.CreateCategories(std::move(kmlDataCollection));
```

so I had an idea to reproduce import-export in this test so we don't use raw gpx for export but import them with LoadKmlFile first.